ntc_templates/templates/alcatel_aos_show_interfaces_ethernet.textfsm

```diff
@@ -0,0 +1,86 @@
+Value PORT (\S+)
+Value STATUS (\S+)
+Value ADMIN_STATUS (\S+)
+Value PORT_DOWN_VIOLATION_REASON (\S.+\S)
+Value LAST_TIME_LINK_CHANGED (\S.+\S)
+Value NUMBER_OF_STATUS_CHANGE (\d+)
+Value PORT_TYPE (\S.+\S)
+Value SFP_XFP (\S.+\S)
+Value INTERFACE_TYPE (\S.+\S)
+Value ETHERNET_POWER_POLICY (\S.+\S)
+Value LINK_QUALITY (\S.+\S)
+Value MAC_ADDRESS (\S.+\S)
+Value BANDWIDTH (\d+)
+Value DUPLEX (\S+)
+Value AUTONEGOTIATION (\S.+\S)
+Value LONG_FRAME_SIZE (\d+)
+Value INTER_FRAME_GAP (\d+)
+Value LOOPBACK_MODE (\S.+\S)
+Value RX_BYTES_RECEIVED (\d+)
+Value RX_BROADCAST_FRAMES (\d+)
+Value RX_UNDERSIZE_FRAMES (\d+)
+Value RX_LOST_FRAMES (\d+)
+Value RX_CRC_ERROR_FRAMES (\d+)
+Value RX_UNICAST_FRAMES (\d+)
+Value RX_MULTICAST_FRAMES (\d+)
+Value RX_OVERSIZE_FRAMES (\d+)
+Value RX_ERROR_FRAMES (\d+)
+Value RX_ALIGNMENT_ERRORS (\d+)
+Value TX_BYTES_TRANSMITTED (\d+)
+Value TX_BROADCAST_FRAMES (\d+)
+Value TX_UNDERSIZE_FRAMES (\d+)
+Value TX_LOST_FRAMES (\d+)
+Value TX_ERROR_FRAMES (\d+)
+Value TX_UNICAST_FRAMES (\d+)
+Value TX_MULTICAST_FRAMES (\d+)
+Value TX_OVERSIZE_FRAMES (\d+)
+Value TX_COLLIDED_FRAMES (\d+)
+Value TX_COLLISIONS (\d+)
+Value TX_LATE_COLLISIONS (\d+)
+Value TX_EXCESSIVE_COLLISIONS (\d+)
+
+
+Start
+# R6
+  ^\s*Slot\/Port\s*${PORT}\s*:\s*$$
+# R8
+  ^\s*Chassis\/Slot\/Port\s*:\s*${PORT}\s*$$
+  ^\s*Operational\s*Status\s*:\s*${STATUS},\s*(?:"(?:${ADMIN_STATUS})?"\s*)?$$
+  ^\s*Port-Down\/Violation\s*Reason\s*:\s*${PORT_DOWN_VIOLATION_REASON},\s*$$
+  ^\s*Last\s*Time\s*Link\s*Changed\s*:\s*${LAST_TIME_LINK_CHANGED}\s*,\s*$$
+  ^\s*Number\s*of\s*Status\s*Change\s*:\s*${NUMBER_OF_STATUS_CHANGE},\s*$$
+  ^\s*Type\s*:\s*${PORT_TYPE},\s*$$
+  ^\s*SFP\/(?:SFP\+\/)?XFP\s*:\s*${SFP_XFP},\s*$$
+  ^\s*Interface\s*Type\s*:\s*${INTERFACE_TYPE},\s*$$
+  ^\s*EPP\s*:\s*${ETHERNET_POWER_POLICY},\s*$$
+  ^\s*Link-Quality\s*:\s*${LINK_QUALITY},\s*$$
+  ^\s*MAC\s*address\s*:\s*${MAC_ADDRESS},\s*$$
+  ^\s*BandWidth\s*\(Megabits\)\s*:\s*(-|${BANDWIDTH})\s*,\s*Duplex\s*:\s*(-|${DUPLEX})\s*,\s*$$
```

Comment:
This seems like an instance where `BandWidth` could later show up as `Bandwidth` (unlikely the B will ever be lowercase for the label).

Yeah this is a bit "future proofed", but I wouldn't suggest it if I have never seen bandwidth written as BandWidth with a capital W. ( :grin: at Alcatel)

Thoughts? @evilmonkey19

```suggestion
  ^\s*Band[Ww]idth\s*\(Megabits\)\s*:\s*(-|${BANDWIDTH})\s*,\s*Duplex\s*:\s*(-|${DUPLEX})\s*,\s*$$
```